### ISSUE TITLE
Multi-interest separation (discoverability) with account grouping master feature

### ISSUE THREAD

**Issue by LeeteqXV** [tootsuite/mastodon#5515]
I am in the middle of examplifying and exploring the benefits, possibilities and pitfalls of multi-account usage, using a mix of personally hosted blogs/web sites, Twitter (Tweetdeck) and a mix of Mastodon instances. (Overview: www.KetilXV.me)

This is because I am sure that most of us cannot cope with all the poor quality information out there, and we need to help each other both navigate, find/boost quality content, and also avoid irrelevant content.
That last part is often our own fault, especially in a Social Media context:

I have a **personal/interest** account at **mastodon.club/@EzqimoXV**, 
one **strategy related** and one **information-overload related** account at **mastodon.host/@FuzboleroXV** and **mastodon.host/@info_overlord**, 
then a technology focused account at **mastodon.technology/@LeeteqXV**, 
- and finally an account about society/politics/general developments at **mastodon.social/@ZeniorXV**.

_Now, my technology focused account over at mastodon.technology would "love" to "follow" the main developer account(s) for Mastodon (Gargron++), BUT, in the case of Gargron, his account contains so many CAT toots/boosts that it would destroy the topic-focused timeline I wish to maintain for my @LeeteqXV account._

Therefore I follow him instead with my personal interests account at mastodon.club (because I too love cats/animals, as it happens), with the side effect that I watch/check it very much less frequent, and miss a lot of mastodon related stuff that are pushed out of "sight" quicker than I can manage to keep on top of it. So, in my "hats"/separation mindset, I would like to be able to follow just those toots that are about technology that comes from Gargron, but the current solution does not provide tools for that.

Fortunately, with the recent multi-account feature updates of the nice **Mastalab Android client** , I can now open toots and boost/favourite them via/from any of my related accounts easily. That is great, but there is much more work to be done in this field still.

I think that a valid challenge for the architecture of systems like Mastodon, is to let various users come in with various interests, focuses and "roles"/"hats", and find tools to conveniently separate them here.

Right now, I am examplifying by establishing multiple accounts, but they are not tied together, there is no "master acccount function" that would tell each instance admin that these accounts "belong together" and represent the same person. 

That could be practical both for security reasons, and in order to innovate along the line of everyone having "multiple hats", including better discoverability, search and filtering.

Not sure how exactly this could be done (should be a broader discussion first), but this post here is meant as a feature request of some sort.

**Comment by trwnh** [comment]
It might be useful to consider the metaphors that other social networks use:

- Tumblr uses tags, and browser extensions like XKit allow you to filter out certain tagged posts from your dashboard. Additionally, RSS feeds exist for each tag on a user's blog. This allows you to follow only certain tagged posts in your RSS feed, but requires the user to tag their posts.
- diaspora* uses aspects, and each aspect allows you to categorize the people you follow. Functionally, you can choose which posts are visible to which aspects. For example, you can make a certain post visible only to the Friends aspect, but not the Family aspect. Aspects also let you filter your stream to see only posts by other users in specific aspects, e.g. seeing posts from Friends but not from Family.
- Google+ uses circles and collections. Circles work like diaspora* aspects. Collections allow you to categorize your own posts. For example, you can put science posts in a Science collection, and photography posts in a Photography collection. Other users can follow your collection without necessarily following your user account.

Between tags, aspects, and collections, I think a mix will be necessary to properly cover the use case here. A user should be able to categorize their own posts, and they should be able to filter other users' posts. Also, they should be able to create lists of users or of their categories. That should allow for disparate interests and perhaps reduce the need to maintain multiple accounts and multiple timelines. As someone who has typically had between 3 and 15 Twitter accounts at any time, it's a nightmare to manage.

**Comment by LeeteqXV** [comment]
Yes, these are good points. IMO there is also the question / challenge of both a functional user interface to the (consumption of the) information itself (including on small screens) and to the administration of the information (also including on/from small screens).

(Furthermore, ideally also automation; this is a perfect use case for todays level of "A.I." should be able to handle nicely, with an interface to manage such automation.)

**Comment by LeeteqXV** [comment]
Regarding tagging/filtering; that is also something that ideally should be discussed more broadly, as so many services are facing the same challenge. From the perspective of a better functioning and efficient society (with citizens wasting less time on unnecessary confusion or extra work) it would be immensely practical the more services agree on the same categorisation scheme (either using the same terminology or synonyms sharing the same logic).

**Comment by trwnh** [comment]
Duplicate of #1208

**Comment by aquariusdev** [comment]
I would also like to see this in Mastodon. Friendica added this sometime around 2020 but then downgraded it to something more subtle. Hubzilla also did this as well. I wish they had better documentation about it because the first time someone even discussed this feature in Friendica was in this recent blog post: https://deadsuperhero.com/2022/02/towards-a-greater-federated-architecture/#:~:text=this%20is%20an%20inverse%20concept%20to%20aspects%2C%20and%20to%20date%2C%20i%E2%80%99ve%20really%20only%20seen%20platforms%20based%20on%20friendica%20do%20this.%20 "This is an inverse concept to Aspects, and to date, I’ve really only seen platforms based on Friendica do this. "
